[2023-07-17T09:44:47.195+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.data_acquisition manual__2023-07-17T09:44:39.213744+00:00 [queued]>
[2023-07-17T09:44:47.204+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.data_acquisition manual__2023-07-17T09:44:39.213744+00:00 [queued]>
[2023-07-17T09:44:47.205+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-17T09:44:47.220+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): data_acquisition> on 2023-07-17 09:44:39.213744+00:00
[2023-07-17T09:44:47.224+0000] {standard_task_runner.py:57} INFO - Started process 1012 to run task
[2023-07-17T09:44:47.227+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'data_acquisition', 'manual__2023-07-17T09:44:39.213744+00:00', '--job-id', '651', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmp1r2cr5uk']
[2023-07-17T09:44:47.230+0000] {standard_task_runner.py:85} INFO - Job 651: Subtask data_acquisition
[2023-07-17T09:44:47.276+0000] {task_command.py:410} INFO - Running <TaskInstance: main_dag.data_acquisition manual__2023-07-17T09:44:39.213744+00:00 [running]> on host a888360a1584
[2023-07-17T09:44:47.371+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='data_acquisition' AIRFLOW_CTX_EXECUTION_DATE='2023-07-17T09:44:39.213744+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-17T09:44:39.213744+00:00'
[2023-07-17T09:47:03.584+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/wikipedia/wikipedia.py:389 GuessedAtParserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("lxml"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

The code that caused this warning is on line 389 of the file /home/***/.local/lib/python3.7/site-packages/wikipedia/wikipedia.py. To get rid of this warning, pass the additional argument 'features="lxml"' to the BeautifulSoup constructor.
[2023-07-17T09:47:03.594+0000] {logging_mixin.py:150} INFO - Could not load Casp
[2023-07-17T09:47:55.389+0000] {logging_mixin.py:150} INFO - Could not load Computer Vision
[2023-07-17T09:48:23.653+0000] {logging_mixin.py:150} INFO - Could not load Dbscan
[2023-07-17T09:48:32.147+0000] {logging_mixin.py:150} INFO - Could not load Data Mining
[2023-07-17T09:51:07.427+0000] {logging_mixin.py:150} INFO - Could not load Github
[2023-07-17T09:51:22.119+0000] {logging_mixin.py:150} INFO - Could not load Google
[2023-07-17T09:52:23.754+0000] {logging_mixin.py:150} INFO - Could not load Identity Matrix
[2023-07-17T09:54:20.652+0000] {logging_mixin.py:150} INFO - Could not load Machine Learning
[2023-07-17T09:55:10.771+0000] {logging_mixin.py:150} INFO - Could not load Multilayer Perceptron
[2023-07-17T09:55:18.653+0000] {logging_mixin.py:150} INFO - Could not load Np-Hard
[2023-07-17T09:56:29.590+0000] {logging_mixin.py:150} INFO - Could not load Pmc (Identifier)
[2023-07-17T09:56:34.901+0000] {logging_mixin.py:150} INFO - Could not load Petar Veličković
[2023-07-17T09:56:37.030+0000] {logging_mixin.py:150} INFO - Could not load Physics
[2023-07-17T09:56:39.493+0000] {logging_mixin.py:150} INFO - Could not load Pixel
[2023-07-17T09:57:15.472+0000] {logging_mixin.py:150} INFO - Could not load Relu
[2023-07-17T09:58:32.307+0000] {logging_mixin.py:150} INFO - Could not load Social Graph
[2023-07-17T09:58:59.314+0000] {logging_mixin.py:150} INFO - Could not load Tensorflow
[2023-07-17T09:59:00.700+0000] {logging_mixin.py:150} INFO - Could not load Thomas Kipf
[2023-07-17T09:59:21.081+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-07-17T09:59:21.092+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=data_acquisition, execution_date=20230717T094439, start_date=20230717T094447, end_date=20230717T095921
[2023-07-17T09:59:21.139+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-17T09:59:21.179+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
